plugins/api-docs/src/copyFunctions.ts

```diff
@@ -0,0 +1,49 @@
+// @ts-ignore
+import fs from "fs-extra";
+import path from "path";
+
+/**
+ * Docusaurus / mdx build can't process empty comments in markdown.
+ *
+ * In MDX 1 there is a behaviour that will lead to links not being rendered as clickable links. See
+ * https://github.com/mdx-js/mdx/issues/1571#issuecomment-853384939
+ *
+ * This causes:
+ *  <b>Implements:</b> [Percentage](./foundation-filters.percentage.md)
+ *  <b>Extends:</b> [ClientFilter](./foundation-filters.clientfilter.md)&lt;[NodeEnvParams](./foundation-filters.nodeenvparams.md)&gt;
+ *  etc.
+ *
+ * ...to render unlinked, so users actually see text like this instead:
+ *
+ * Implements: [NodeEnv](/web/filters/docs/api/foundation-filters.nodeenv)
+ *
+ * Replacing html tags like <b> with their markdown equivalent `**` fixes the issue, as the line starts with markdown.
+ */
+function cleanseMarkdownContent(input: string) {
+  return input.replace(/<!-- -->/g, "").replace(/<b>|<\/b>/g, "**");
```

Comment:
Should we be doing this on the generation side?
Works fine for now :)

Reply:
Yeah I think the fact that the docs are full of empty markdown comments is probably a bug with their docs generation tools 😂 there is an existing ticket somewhere to migrate to their newer version of the tool which can handle different OS line endings better too. 
But for now we'll have to leave it here.